migrations/20250701153637-move-file-upload-status-to-files.cjs

```diff
@@ -0,0 +1,95 @@
+/* eslint-disable */
+module.exports = {
+  /**
+   * Safely copies documents from 'file-upload-status' to 'files' collection
+   * @param db {import('mongodb').Db}
+   * @param client {import('mongodb').MongoClient}
+   * @returns {Promise<void>}
+   */
+  async up(db, client) {
+    const filesCollection = db.collection('files')
+    const fileUploadStatusCollection = db.collection('file-upload-status')
+
+    const documentsToMigrate = await fileUploadStatusCollection
```

Comment:
How many records could be returned here? If it's a lot - maybe we need some pagination or use cursors?